packages/py-moose-lib/moose_lib/internal.py

```
@@ -566,6 +596,40 @@ class InfrastructureMap(BaseModel):
+    Returns:
+        The duration in seconds.
+
+    Raises:
+        ValueError: If the unit is not recognized.
+    """
+    multiplier = _UNIT_TO_SECONDS.get(unit)
+    if multiplier is None:
+        raise ValueError(
+            f'Unknown time unit: "{unit}". '
+            f"Supported units: {', '.join(_UNIT_TO_SECONDS.keys())}"
+        )
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)